container-images/pragmatic/build_pragmatic.sh

```diff
@@ -27,14 +27,20 @@ install_pytorch() {
 }
 
 clone_and_build_pragmatic() {
-if [ "$2" == "docling" ]; then
-   ${PYTHON_VERSION} pip install docling --extra-index-url https://download.pytorch.org/whl/$1
-fi
   git clone https://github.com/redhat-et/PRAGmatic
   cd PRAGmatic
   git submodule update --init --recursive
 
-  ${PYTHON_VERSION} pip install -r requirements.txt --prefix=/usr
+  if [ "$2" == "docling" ]; then
+      ${PYTHON_VERSION} pip install docling --extra-index-url https://download.pytorch.org/whl/$1
+      ${PYTHON_VERSION} pip install -r requirements.txt --prefix=/usr
+  else
+      ${PYTHON_VERSION} pip install torch --extra-index-url https://download.pytorch.org/whl/$1
+      tmpfile=$(mktemp)
+      grep -v docling requirements.txt > ${tmpfile}
+      ${PYTHON_VERSION} pip install -r /tmp/nodocling.txt --prefix=/usr
```

Comment:
@rhatdan when this code is executed:

https://github.com/containers/ramalama/actions/runs/13356993323/job/37301130079?pr=831

it complains with this:

+ grep -v docling requirements.txt
+ python3 -m pip install -r /tmp/nodocling.txt --prefix=/usr
ERROR: Could not open requirements file: [Errno 2] No such file or directory: '/tmp/nodocling.txt'
Error: building at STEP "RUN sh /tmp/build_pragmatic.sh ${GPU} ${CONTENT}": while running runtime: exit status 1
Error: Process completed with exit code 1.